test/go-tests/test_importer_test.go

```diff
@@ -0,0 +1,163 @@
+package go_tests
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/stretchr/testify/require"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+)
+
+const importerShipyard = `--- 
+apiVersion: "spec.keptn.sh/0.2.0"
+kind: "Shipyard"
+metadata:
+  name: "shipyard"
+spec:
+  stages:
+    - name: "dev"
+      sequences:
+        - name: "delivery"
+          tasks:
+            - name: "deployment"
+              properties:
+                deploymentstrategy: "user_managed"`
+
+func Test_ImportCorrectManifest(t *testing.T) {
+	projectName := "keptn-importer-test"
+	serviceName := "my-service-name"
+
+	t.Logf("Creating a new project %s with Gitea Upstream", projectName)
+	shipyardFilePath, err := CreateTmpShipyardFile(importerShipyard)
+	require.Nil(t, err)
+	projectName, err = CreateProject(projectName, shipyardFilePath)
+	require.Nil(t, err)
+
+	// Convert folder to Zip
+	err = createZipFileFromDirectory("../assets/import/sample-package/", "./sample-package.zip", false)
+	require.Nil(t, err)
+
+	// Getting Keptn API credentials
+	_, keptnApiUrl, err := GetApiCredentials()
+	require.Nil(t, err)
+
+	// Make API call with ZIP file
+	responseCode, err := ImportUploadZipToProject(fmt.Sprintf("%s/v1/import", keptnApiUrl), projectName, "./sample-package.zip")
+	require.Nil(t, err)
+	require.Equal(t, 200, responseCode, fmt.Sprintf("Expected response status 200 but got %d", responseCode))
+
+	// Check if service exists
+	approvalCLIOutput, err := ExecuteCommand(fmt.Sprintf("keptn get service --project %s %s", projectName, serviceName))
```

Comment:
This variable is weirldy named `approvalCLIOutput`, but this has nothing to do with approvals...

Reply:
Probably a copy-paste error. It should be fixed now.